PurchasesCoreSwift/SubscriberAttributes/SubscriberAttribute.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  SubscriberAttribute.swift
+//  PurchasesCoreSwift
+//
+//  Created by Joshua Liebowitz on 7/1/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCSubscriberAttribute) public class SubscriberAttribute: NSObject {
+
+    @objc public static let keyKey = "key"
+    @objc public static let valueKey = "value"
+    @objc public static let setTimeKey = "setTime"
+    @objc public static let isSyncedKey = "isSynced"
+
+    static let backendValueKey = "value"
+    static let backendTimestampKey = "updated_at_ms"
+
+    public let setTime: Date
```

Comment:
surprised this one doesn't need `@objc`, is it only used by stuff that's migrated? if so, maybe we could already make it internal?

Reply:
It can be private, it's only referenced in the file.
Updated!

Reply:
Oyyy actually in the `PurchasesTest` target, it is referenced. So for now, we'll keep it public but also as part of the `#todo` so we can change it when we are done.

Reply:
I see. keeping it `internal` should also work since we do `@testable import PurchasesCoreSwift` for tests 👍

Reply:
Already tried that, the tests are referencing the class through `Purchases` so it's one level nested @testable seems to only work if the target is directly linked instead of indirectly.

Reply:
Right. We should be moving tests over to `PurchasesCoreSwiftTests` when we move corresponding classes, and updating the `import Purchases` statements to `@testable import PurchasesCoreSwift`, [like here ](https://github.com/RevenueCat/purchases-ios/pull/608/files#diff-6e795e719a9b41fcd5735980fd2532f93d255a731e79dffe8e3af58edadbeb10R14)

Reply:
Yeah, definitely a great idea. I've updated the test and moved it over. We still need `public` until the other pieces get turned into swift. Specifically `RCDeviceCache` and  `RCSubscriberAttributesManager`, and `RCBacked`